types.d.ts

```diff
@@ -0,0 +1,10 @@
+// This TypeScript modules definition file is generated by vaadin-maven-plugin.
```

Comment:
This file was intentionally deleted earlier so it does not get out of date

Reply:
how about add it into the `.gitignore` then.

Reply:
That sounds very strange as the comment in the file says "It is recommended to commit this file to the VCS."